src/background.js

```diff
@@ -53,7 +53,16 @@ async function switch_tab(back_num) {
 chrome.commands.onCommand.addListener(async function(command) {
   // the 'switch-tab' command is defined in the manifest
   if (command === "switch-tab") {
-      await switch_tab(1);
+      console.log("Chrome.commands: Trying switch");
+      let promisedTabs = chrome.tabs.query({active: true, currentWindow: true});
+      promisedTabs.then(async (tabs) => {
+        let url = tabs[0].url;
+        const toMatch = "chrome://";
+        if (url.slice(0,9) === toMatch) {
+          console.log("       Restricted Switch Done");
+          await switch_tab(1);
+        }      
+      });
```

Comment:
I'm pretty sure you can just do this:
```suggestion
      let tabs = await chrome.tabs.query({active: true, currentWindow: true});
      let url = tabs[0].url;
      const toMatch = "chrome://";
      if (url.slice(0,9) === toMatch) {
        console.log("       Restricted Switch Done");
        await switch_tab(1);
      } 
```

Reply:
This isn't super important, more of a nit bc it makes the code look nicer. The main thing I wanted to see were some tests or a test plan for the future.

Reply:
You're right, this works. I changed the code to reflect your suggestion.